schemas/Product.yaml

```diff
@@ -1,20 +1,154 @@
-required:
+# copyright 2021 Lucid Green, Inc.
+
+required: 
   - id
   - name
-  - strainId
-  - strainName
-  - productCategoryName
-  - productCategoryType
-  - quantityType
-  - publicIngredients
-  - description
-  - isUsed
+  - updated_on
+  - product_type
+  - usage
+  - variety
+  - package_size
```

Comment:
Same with package_size / usage, I think type and variety are necessary but these seem like specific values that a product may not necessarily contain (based on the input from the brand / label vendor)

Reply:
ok